pyproject.toml

```diff
@@ -120,6 +120,7 @@ pytest = "^7.3.0"
 pytest-cov = "^4.0.0"
 pytest-dotenv = "^0.5.2"
 duckdb-engine = "^0.7.0"
+fauna= "0.6.1"
```

Comment:
Only packages that help with testing infrastructure go in the test group. 

```suggestion
```

See information here on how to introduce a new optional dependency: https://github.com/hwchase17/langchain/blob/master/.github/CONTRIBUTING.md#working-with-optional-dependencies